Content.Shared/CCVar/CCVars.cs

```diff
@@ -1242,7 +1242,7 @@ public sealed class CCVars : CVars
         ///     Config for when the restart vote should be allowed to be called based on percentage of ghosts.
         ///
         public static readonly CVarDef<int> VoteRestartGhostPercentage =
-            CVarDef.Create("vote.restart_ghost_percentage", 75, CVar.SERVERONLY);
+            CVarDef.Create("vote.restart_ghost_percentage", 50, CVar.SERVERONLY);
```

Comment:
Talked to admins and they felt this was more appropiate
```suggestion
            CVarDef.Create("vote.restart_ghost_percentage", 55, CVar.SERVERONLY);
```